docs/calico-with-docker/without-docker-networking/README.md

```diff
@@ -23,6 +23,9 @@ with a number of installation dependencies.
 
 Follow the instructions in one of the tutorials below to set up a virtualized
 environment using Vagrant or a cloud service - be sure to follow the
+
+[NJ] Why would I want a 'virtualized environment', and what does that mean?  My situation is that I already have a Linux host, and that I want to use Docker to create some containers on it, and use Calico to network those.  Is there some 'environment' that I need that I do not already have?
+
```

Comment:
No, you should use the manual setup instructions just below.

I think this is the correct ordering as well; most people who come to these instructions are going to use them on VMs.